src/components/pages/module-form/useResetFormOnCloseModal.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useRef } from 'react';
+
+export const useResetFormOnCloseModal = ({ resetFields, visible }) => {
```

Comment:
I love your use of a custom hook here! Great way to encapsulate reusable functionality. Great job!